### ISSUE TITLE
should be UI scale in settings

### ISSUE THREAD

**Issue by lzk228** [space-wizards/SS14.Launcher#122]

**Comment by PJB3005** [comment]
Is this about changing the in-game UI scale in case somebody breaks it? Or is this for changing the UI scale of the launcher itself?

**Comment by lzk228** [comment]
for launcher itself, because buttons are too small
![image](https://github.com/space-wizards/SS14.Launcher/assets/124214523/21927797-0e5b-4d4f-b134-17d1a18e87dd)
i think if someone playing on 4k monitors its even worse looking

**Comment by PJB3005** [comment]
Yeah that's just a Linux-specific issue with it not supporting proper DPI scaling (and now that Wayland finally does, Avalonia doesn't support it yet). On a competent OS this works fine.

**Comment by simplepad** [comment]
Workaround described here: https://github.com/AvaloniaUI/Avalonia/wiki/Configuring-X11-per-monitor-DPI

Basically, put something like
`AVALONIA_SCREEN_SCALE_FACTORS='eDP-1=2' %command%`
in your launch options on steam, except replace `eDP-1` with the name of your monitor (can be obtained using `xrandr --listactivemonitors`). Adjust the number after the equals sign to your prefered scaling factor. This works for me on gnome wayland.

**Comment by lzk228** [comment]
> Basically, put something like
> `AVALONIA_SCREEN_SCALE_FACTORS='eDP-1=2' %command%`
> in your launch options on steam

not everyone plays via steam, so the scale option still should be added

> (and now that Wayland finally does, Avalonia doesn't support it yet).

@PJB3005 could you please look if it's possible to make the scale option now?
